browser/help/topics/v3-ht-annotations.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: v3-ht-annotations
```

Comment:
not sure if this needs to be renamed (this is is a question for someone on the browser team)